### ISSUE TITLE
Failed to load library libonnxruntime_providers_cuda.so with error: libonnxruntime_providers_cuda.so: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by willmakley** [aws/aws-toolkit-vscode#5868]
## Problem

On opening a Ruby on Rails project in VSCode, Amazon Q Language Server shows the following error:

```
Request initialize failed with message: /onnxruntime_src/onnxruntime/core/session/provider_bridge_ort.cc:1637 onnxruntime::Provider& onnxruntime::ProviderLibrary::Get() [ONNXRuntimeError] : 1 : FAIL : Failed to load library libonnxruntime_providers_cuda.so with error: libonnxruntime_providers_cuda.so: cannot open shared object file: No such file or directory
```

## Steps to reproduce the issue

Open a project in VSCode

## Expected behavior

No error.

## System details (run `AWS: About` and/or `Amazon Q: About`)

-   OS: Ubuntu 24 running on WSL 2, all latest updates
-   Visual Studio Code version: 1.94.2
-   AWS Toolkit version:  N/A
-   Amazon Q version: 1.30.0 (I think) last updated 2024-10-22, 10:49:44

**Comment by leigaol** [comment]
Hi @willmakley , in your Amazon Q VS Code settings, do you have the Amazon Q: Workspace Index Use GPU enabled? If so, is it possible for you to turn it off and then restart or reload VS Code? Thank you!

**Comment by willmakley** [comment]
Yes, it was on (by default) and turning it off does seem to have resolved the issue.

**Comment by leigaol** [comment]
On by default is not expected. In our configuration, we make it off by default. https://github.com/aws/aws-toolkit-vscode/blob/master/packages/amazonq/package.json#L157 

You are seeing this error because CUDA driver was not ready in the Ubuntu environment. If there is a valid CUDA installed, the error should go away.
